Client/Configuration/AdHoc.xcconfig

```diff
@@ -1,7 +1,11 @@
+// Bundle display name
 BUNDLE_DISPLAY_NAME = $(TARGET_NAME) β
 
 // Bundle Identifier
 APP_BUNDLE_ID = $(PRODUCT_BUNDLE_IDENTIFIER).nightly
 
 // URL Scheme
 CLIQZ_URL_SCHEME = $(TARGET_NAME)Nightly
+
+// Bundle short version
+CLIQZ_BUNDLE_SHORT_VERSION = 3.1.0
```

Comment:
I believe the version should stay on the target level. Ghostery will definitely have different version from Cliqz.